src/pycea/tl/topology.py

```diff
@@ -0,0 +1,107 @@
+import networkx as nx
+import treedata as td
+from scipy.special import comb as nCk
+
+from pycea.utils import (
+    get_root,
+    get_trees,
+)
+
+
+def compute_expansion_pvalues(
```

Comment:
compute_expansion_pvalues -> expansion_pvalues . Let's remove compute from the function name since it's annoying to type and does not provide additional information about what the function does.